server/game/cards/03-WC/CityStateInterest.js

```diff
@@ -7,7 +7,16 @@ class CityStateInterest extends Card {
             gameAction: ability.actions.capture((context) => ({
                 target: context.player.creaturesInPlay
             })),
-            effect: 'make each friendly creature capture 1 amber'
+            effect: 'to capture 1 amber onto each friendly creature',
```

Comment:
Any way to turn off a message so just the then: messages? Or a way to move the then message up and have access to the preThenEvents info?